doc/api/fs.markdown

```diff
@@ -1157,6 +1157,17 @@ reliably or at all.
 You can still use `fs.watchFile`, which uses stat polling, but it is slower and
 less reliable.
 
+#### Inodes
+
+<!--type=misc-->
+
+On Linux and OS X systems, `fs.watch()` resolves the watched path to an
+[inode][]. If the watched path is deleted and then created so that it has a new
+inode, an event for the delete will be emitted but the watch will remain
```

Comment:
Nit... Perhaps: `When a watched path is deleted and then recreated, it is assigned a new [inode][]. A notification for the deletion event will be emitted but the watch will continue on the *original* inode. Events for the new inode will not be emitted. This is expected behavior.`